admin/allow-services.rst

```diff
@@ -47,36 +52,65 @@ The following examples show how to set the ``HTTP_PROXY`` and ``HTTPS_PROXY`` en
 
 .. tabs::
 
-   .. code-tab:: powershell Windows
+   .. code-tab:: bash Linux (Systemd)
+
+      # Add proxy settings to the environment for the installer script
+
+      sudo sed -i "$ a http_proxy=http://<proxy.address:port>" /etc/environment
+      sudo sed -i "$ a https_proxy=http://<proxy.address:port>" /etc/environment
+      sudo sed -i "$ a no_proxy=<address>" /etc/environment
```

Comment:
These `sed` commands will not work if `/etc/environment` is empty.  I think it may be safer to use a heredoc, e.g.
```suggestion
      cat <<EOF | sudo tee -a /etc/environment
      NO_PROXY=<address,anotheraddress>
      HTTP_PROXY=http://<proxy.address:port>
      HTTPS_PROXY=http://<proxy.address:port>
      EOF
```
And after `/etc/environment` is updated, the user will need to restart their shell (log out and log back in) for these env vars to take effect.